libsolidity/interface/CompilerStack.h

```diff
@@ -419,10 +427,14 @@ class CompilerStack: public langutil::CharStreamProvider, public evmasm::Abstrac
 		util::LazyInit<Json const> devDocumentation;
 		util::LazyInit<Json const> generatedSources;
 		util::LazyInit<Json const> runtimeGeneratedSources;
+		util::LazyInit<Json const> ethdebug;
+		util::LazyInit<Json const> ethdebugRuntime;
```

Comment:
Please not. Even storing Yul ASTs was making memory usage go through the roof and giving us a 25% performance hit. And I expect that the whole ethdebug dump will be much bigger than the AST...

See https://github.com/ethereum/solidity/pull/15451#issuecomment-2371706002.

Reply:
yeah I saw that and I agree that this might get quite bad with the ethdebug stuff..